packages/discount-code-exporter/package.json

```diff
@@ -20,7 +20,25 @@
     "Williams Omayuku <williams.omayuku@commercetools.com>"
   ],
   "main": "lib/index.js",
+  "bin": {
+    "discount-code-exporter": "bin/discount-code-exporter.js"
```

Comment:
#userfriendly... I think it is important that we make this decision __once__ so we don't deviate on future packages